portfolio/src/main/webapp/style.css

```diff
@@ -2,8 +2,9 @@
   margin-left: auto;
   margin-right: auto;
   width: 650px;
+  max-width: 60em;
```

Comment:
nit: CSS best practice is to alphabetize properties

Reply:
Fixed!